.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+Fixes #ISSUE.
+
+
+## Pre-merge checklist
+
+<!--- A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
+
+- [ ] Docs
+   - `docs/` have been added/updated **OR**
+   - No updates to the docs necessary after careful consideration.
+
+- [ ] QC
+   - `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
+   -  No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary.
+
+- [ ] Reviewed
+   - Has been sufficiently reviewed by at least one review from a different team member of the Mondo Technical team.
+
+## High-level description
```

Comment:
```suggestion
## Overview
```
Don't have to make this change, but this is the header I use at the start of most of my issues and PRs, and is essentially synonymous.

More importantly, I suggest re-ordering like this:
1. "Fixes #ISSUE"
2. Overview
3. Checklist

Reply:
Feel free to edit the file to apply your suggestion (not edit mode, just go to my branch and edit the file as you see fit). You have good ideas, I am sure I will agree!